ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -298,6 +337,59 @@ const triggerSet: OopsyTriggerSet<Data> = {
           delete data.partyMitTracker[abilityId];
       },
     },
+    {
+      id: 'General Overwritten Damage Effect',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: targetDamageAbilityIds.concat(partyDamageAbilityIds) }),
+      mistake: (data, matches) => {
+        const isTargetDamage = targetDamageAbilityIds.includes(matches.id);
+        const isPartyDamage = partyDamageAbilityIds.includes(matches.id);
+        if (isTargetDamage && matches.targetId === 'E0000000') // missed
+          return;
+        if (isPartyDamage && !data.party.partyIds_.includes(matches.sourceId))
+          return;
```

Comment:
If we only check party membership for `isPartyDamage` and not `isTargetDamage`, I'm wondering how noisy this could get in overworld content (boss fates, hunts, etc.) where you may have lots of overwriting going on.  I don't think we have a good way to zone-filter this though.

Do you think it would make sense to also check party membership (maybe alliance membership too, just to be safe) for target damage effects?  Or does that run the risk of knocking out mistakes that we do actually want reported?

Reply:
This is intentional. Target effects like Chain Stratagem benefit the entire alliance (across parties), and also get overwritten across parties, in the same manner as effects like Reprisal. It may get spammy in hunt trains (oopsy is going to get spammy in general there), but the thinking was more for the upcoming 24-man savage or BLU alliance raids, where coordinating target effects across parties and not overwriting each other is more important.

Reply:
Yeah, makes sense.  I'm okay with leaving as you've got it for now.  If overworld stuff gets too spammy, we can always revisit.